tests/show_techsupport/test_techsupport.py

```diff
@@ -0,0 +1,515 @@
+import pytest
+import os
+import pprint
+from loganalyzer import LogAnalyzer, LogAnalyzerError
+from common import reboot, port_toggle
+import pdb
+import optparse
+import time
+from random import randint
+import logging 
+logger = logging.getLogger(__name__)
+
+LOOP_RANGE = 10
+LOOP_DELAY=10
+SINCE=0
+rules_list = ['acl_rules']
+teardown_list = []
+
+
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+DUT_TMP_DIR = os.path.join('tmp', os.path.basename(BASE_DIR))
+FILES_DIR = os.path.join(BASE_DIR, 'files')
+TEMPLATE_DIR = os.path.join(BASE_DIR, 'templates')
+
+ACL_TABLE_TEMPLATE = 'acltb_table.j2'
+ACL_RULES_FULL_TEMPLATE = 'acltb_test_rules.j2'
+ACL_RULES_PART_TEMPLATES = tuple('acltb_test_rules_part_{}.j2'.format(i) for i in xrange(1, 3))
+ACL_REMOVE_RULES_FILE = 'acl_rules_del.json'
+ACL_RULE_PERSISTENT_FILE = 'acl_rule_persistent.json'
+ACL_RULE_PERSISTENT_DEL_FILE = 'acl_rule_persistent-del.json'
+ACL_RULE_PERSISTENT_J2 = 'acl_rule_persistent.json.j2'
+
+pytestmark = [
+    pytest.mark.acl,
+    pytest.mark.disable_loganalyzer  # disable automatic loganalyzer
+]
+
+
+#########################################
+############### ACL PART ################
+#########################################
+
+
+DST_IP_TOR = '172.16.1.0'
+DST_IP_TOR_FORWARDED = '172.16.2.0'
+DST_IP_TOR_BLOCKED = '172.16.3.0'
+DST_IP_SPINE = '192.168.0.0'
+DST_IP_SPINE_FORWARDED = '192.168.0.16'
+DST_IP_SPINE_BLOCKED = '192.168.0.17'
+
+LOG_EXPECT_ACL_TABLE_CREATE_RE = '.*Created ACL table.*'
+LOG_EXPECT_ACL_TABLE_REMOVE_RE = '.*Successfully deleted ACL table.*'
+LOG_EXPECT_ACL_RULE_CREATE_RE = '.*Successfully created ACL rule.*'
+LOG_EXPECT_ACL_RULE_REMOVE_RE = '.*Successfully deleted ACL rule.*'
+
+
+def setup_acl_rules(duthost, acl_setup, acl_table):
+        """
+        setup rules on DUT
+        :param dut: dut host
+        :param setup: setup information
+        :param acl_table: acl table creating fixture
+        :return:
+        """
+        name = acl_table['name']
+        dut_conf_file_path = os.path.join(acl_setup['dut_tmp_dir'], 'acl_rules_{}.json'.format(name))
+
+        logger.info('generating config for ACL rules, ACL table {}'.format(name))
+        duthost.template(src=os.path.join(TEMPLATE_DIR, ACL_RULES_FULL_TEMPLATE),
+                     dest=dut_conf_file_path)
+
+        logger.info('applying {}'.format(dut_conf_file_path))
+        duthost.command('config acl update full {}'.format(dut_conf_file_path))
+
+
+@pytest.fixture(scope='function')
+def acl_rules(duthost, localhost, acl_setup, acl_table):
```

Comment:
fixutures are copy-pasted from acl test. You don't need to have all of them and the way they are structured is for acl test but not for this one.

I suggest to create a single fixture that will create ACL rules in DATA ACL table. No need to create a seperate table.

Reply:
done